reactioncommerce/reaction-docs-static
# (fix): correct text for lint and test commands

Resolves: Resolves unreported issue
Impact: **minor**  
Type: **docs**

## Issue
Descriptions for the `npm run lint` and `reaction test` were backwards.

## Solution
Switched descriptions

## Breaking changes
None

## Testing
1. Visit the main "Contributing Guide" page -> "Pass all tests"
2. Confirm the description for `npm run lint` reads "Pass linter code review and follow Reaction style guidelines".
3. Confirm the description for `reaction test` reads "Pass current continuous integration tests"
